demos/interactive_face_detection_demo/interactive_face_detection.hpp

```diff
@@ -55,80 +48,18 @@ static const char target_device_message_lm[] = "Optional. Target device for Faci
 "(the list of available devices is shown below). Default value is CPU. Use \"-d HETERO:<comma-separated_devices_list>\" format to specify HETERO plugin. " \
 "The demo will look for a suitable plugin for device specified.";
 
-/// @brief Message for the maximum number of simultaneously processed faces for Age Gender network
-static const char num_batch_ag_message[] = "Optional. Number of maximum simultaneously processed faces for Age/Gender Recognition network " \
-"(by default, it is 16)";
-
-/// @brief Message for the maximum number of simultaneously processed faces for Head Pose network
-static const char num_batch_hp_message[] = "Optional. Number of maximum simultaneously processed faces for Head Pose Estimation network " \
-"(by default, it is 16)";
-
-/// @brief Message for the maximum number of simultaneously processed faces for Emotions network
-static const char num_batch_em_message[] = "Optional. Number of maximum simultaneously processed faces for Emotions Recognition network " \
-"(by default, it is 16)";
-
-/// @brief Message for the maximum number of simultaneously processed faces for Facial Landmarks Estimation network
-static const char num_batch_lm_message[] = "Optional. Number of maximum simultaneously processed faces for Facial Landmarks Estimation network " \
-"(by default, it is 16)";
-
-/// @brief Message for dynamic batching support for AgeGender net
-static const char dyn_batch_ag_message[] = "Optional. Enable dynamic batch size for Age/Gender Recognition network";
-
-/// @brief Message for dynamic batching support for HeadPose net
-static const char dyn_batch_hp_message[] = "Optional. Enable dynamic batch size for Head Pose Estimation network";
-
-/// @brief Message for dynamic batching support for Emotions net
-static const char dyn_batch_em_message[] = "Optional. Enable dynamic batch size for Emotions Recognition network";
-
-/// @brief Message for dynamic batching support for Facial Landmarks Estimation network
-static const char dyn_batch_lm_message[] = "Optional. Enable dynamic batch size for Facial Landmarks Estimation network";
```

Comment:
So these are the options we miss. Please keep this options in the app, and mark it as TODO

Reply:
These options are marked

Reply:
Ok